data/aliases.json

```diff
@@ -0,0 +1,8 @@
+[
+  ["\\mathfrak", "\\frak"],
+  ["\\mathcal", "\\cal"],
+  ["\\mathbb", "\\bb"],
+  ["\\mathbf", "\\bf"],
+  ["\\dfrac", "\\frac"],
+  ["\\ldots", "\\dots"]
+]
```

Comment:
It's been some time since I've worked with LaTeX, but I remember there being a lot of aliases. Does this open up a can of worms? Also, what source are you using for these?